app/services/player.js

```diff
@@ -96,6 +96,13 @@ export default Service.extend({
 		});
 	},
 
+	onMediaNotAvailable(event) {
```

Comment:
Do we need a check to only save if owner of track here? I assume it's called for any track playing.

Reply:
done